src/goto-symex/goto_symex_state.h

```diff
@@ -21,6 +21,8 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "symex_target.h"
 
+#define SYMEX_CONTINUATION_CHECK "symex_continuation_check"
```

Comment:
I would suggest to add an irep_id ID_C_symex_... (not sure about the "continuation" part, see above) as this will inevitably end up in the string table.